nutkit/frontend/driver.py

```diff
@@ -103,13 +103,15 @@ def send_and_receive(self, req, timeout=None, hooks=None, *,
 
     def execute_query(self, cypher, *, params=None, routing=None,
                       database=None, impersonated_user=None,
-                      bookmark_manager=..., tx_meta=None, timeout=None):
+                      bookmark_manager=..., tx_meta=None, timeout=None,
+                      auth=None):
         config = {
             "routing": routing,
             "database": database,
             "impersonatedUser": impersonated_user,
             "txMeta": tx_meta,
             "timeout": timeout,
+            "auth": auth,
```

Comment:
It might be worth matching the name with the one in the `NewSession` request, which is `authorizationToken`.

On a separate note, just wanted to mention that in the Java driver this is not part of the query config.

Reply:
Thanks for the feedback @injectives I'll get this updated!